### ISSUE TITLE
Reprojected face region

### ISSUE THREAD

**Issue by xinwen-cs** [microsoft/Deep3DFaceReconstruction#53]
Hi,
    You have done a grat work! I am confused about **reprojected face region** in 4.1.1 of your paper. Could you please explain it a little bit more?
Thanks

**Comment by YuDeng** [comment]
Hi, the reprojected face region is just the region of the face in the rendered image, which can be directly obtained via the rendering process.

**Comment by Ivy147** [comment]
> Hi, the reprojected face region is just the region of the face in the rendered image, which can be directly obtained via the rendering process.
Hi，yudeng，do you mean the pix_mask obtained via the rendering process?
**pixel_mask = tf.cast(tf.reduce_any(diffuse_colors >= 0, axis=3), tf.float32)**
But I find that the pixel_mask is upside down while the render_image isn't , do I need to turn the pixel_mask myself or is there any settings I ingored?
I would appreciate it if you could give me some advices.

**Comment by Ivy147** [comment]
> Hi，yudeng，do you mean the pix_mask obtained via the rendering process?
> pixel_mask = tf.cast(tf.reduce_any(diffuse_colors >= 0, axis=3), tf.float32)
> But I find that the pixel_mask is upside down while the render_image isn't , do I need to turn the pixel_mask myself or is there any settings I ingored?
> I would appreciate it if you could give me some advices.



> Hi, the reprojected face region is just the region of the face in the rendered image, which can be directly obtained via the rendering process.

Hi，yudeng，do you mean the pix_mask obtained via the rendering process?
pixel_mask = tf.cast(tf.reduce_any(diffuse_colors >= 0, axis=3), tf.float32)
But I find that the pixel_mask is upside down while the render_image isn't , do I need to turn the pixel_mask myself or is there any settings I ingored?
![render](https://user-images.githubusercontent.com/17442416/71648583-457a7980-2d41-11ea-9b60-5e2078d08a10.jpg)
![mask](https://user-images.githubusercontent.com/17442416/71648586-4ca18780-2d41-11ea-90d6-8fd49a7a0147.jpg)
Also，how can I get the 2D pixel position of 3d landmarks?
I would appreciate it if you could give me some advices.
